### DISCUSSION TITLE
Configure restore path

### DISCUSSION THREAD

**Discussion by infirit** [mtkennerly/ludusavi#283] [Q&A]
I am looking at some old(ish) backups that point to my old user folder. I looked in the various yaml files and the UI but I can't find any way to update this path. Where can I change this?

**Comment by mtkennerly** [comment]
Hi! On the "other" screen, you can configure a redirect so that they will be restored to a new location. For example:

> <img width="745" alt="image" src="https://github.com/mtkennerly/ludusavi/assets/4934192/7b601a17-51d8-46e1-b272-68ba52c5f120">
